Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -1544,14 +1544,29 @@ namespace OpenInfraPlatform {
 						carve::geom3d::Vector& first = loop.front();
 						carve::geom3d::Vector& last = loop.back();
 
+						//bool areEqual = GeometrySettings::areEqual(loop.front(), loop.back());
+						if (GeometrySettings::areEqual(loop.front(), loop.back()))

```

Comment:
```suggestion
						if (GeomSettings()->areEqual(loop.front(), loop.back()))
```
Right now VS  refuse to compile this. Please implement these changes, so it could work

Reply:
Resolved: 77684d8